scripts/ecdsa/eigen/update_quorum.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# -x echos all lines for debug
+# set -x
+
+set -o errexit -o nounset -o pipefail
+command -v shellcheck >/dev/null && shellcheck "$0"
+
+export FOUNDRY_DISABLE_NIGHTLY_WARNING=1
+
+# Function to display error message and exit
+error_exit() {
+    echo "Error: $1" >&2
+    exit 1
+}
+
+SCRIPT_DIR="$(realpath "$(dirname "$0")")"
+# shellcheck source=./helpers.sh
+# shellcheck disable=SC1091
+source "$SCRIPT_DIR"/helpers.sh
+
+# Read the deployer private key from file
+if [ -f "$HOME/.nodes/deployer" ]; then
+    deployer_private_key=$(cat "$HOME/.nodes/deployer")
+    echo "Read deployer key from file."
+    deployer_address=$(cast wallet address "$deployer_private_key")
+    echo "Deployer address: $deployer_address"
+else
+    error_exit "Deployer key file not found at $HOME/.nodes/deployer"
+fi
+
+if [ "$DEPLOY_ENV" = "TESTNET" ]; then
+    LOCAL_ETHEREUM_RPC_URL="$TESTNET_RPC_URL"
+    if [ -z "$LOCAL_ETHEREUM_RPC_URL" ]; then
+        error_exit "TESTNET_RPC_URL environment variable is not set"
+    fi
+else
+    LOCAL_ETHEREUM_RPC_URL=${LOCAL_ETHEREUM_RPC_URL:-http://localhost:8545}
+    wait_for_ethereum
+    cast rpc anvil_setBalance $deployer_address 0x10000000000000000000 -r $LOCAL_ETHEREUM_RPC_URL > /dev/null 2>&1 || error_exit "Failed to set balance for deployer"
```

Comment:
I would only do this if there is no balance. But I guess it doesn't really matter.

As a follow up (maybe done in other PR?) maybe you could pull this if-else block, which is used many places, into some helper script. (there is helpers.sh, which can probably be heavily trimmed and added to)